src/js/fetchTimeline.js

```diff
@@ -1,8 +1,15 @@
-const fetchTimeline = () => {
+const fetchTimeline = (timelineType) => {
 
-    let emptyPromise = new Promise((resolve) => resolve([]));
+    const timelinePaths = {
+        home: "http://localhost:8080/api/1.0/twitter/timeline",
+        user: "http://localhost:8080/api/1.0/twitter/user-timeline"
+    }
+
+    let emptyPromise = new Promise((resolve) => resolve(""));
+
+    console.log(timelinePaths[timelineType]);
     
-    return fetch("http://localhost:8080/api/1.0/twitter/timeline")
+    return fetch(timelinePaths[timelineType])
```

Comment:
same comment as in the backend, separate out instead of conditionally doing different items